Automattic/wp-calypso
# Add VideoPress "benefit" card to plan removal screen

#### Proposed Changes

* Add details about how many videos will be not accessible when removing a Jetpack plan when VideoPress is used

Jetpack VideoPress Product
<img width="1197" alt="Screen Shot 2022-08-31 at 9 47 11 AM" src="https://user-images.githubusercontent.com/4081020/187701839-400a9918-91ac-407b-a737-ee72ba584217.png">


Jetpack Complete (includes VideoPress)
<img width="1202" alt="Screen Shot 2022-08-31 at 9 46 48 AM" src="https://user-images.githubusercontent.com/4081020/187701864-c5e1376b-5964-4679-960f-b4b666d3df9a.png">



#### Testing Instructions



* Pre-testing: Create a new site with a VideoPress plan and upload a few videos to it (if you don't already have one)
* visit `/me/purchases/`
* select your Jetpack site
* Click "Remove [Jetpack Plan]" button
* The modal that appears should include a Video section with the correct VideoPress video count
* Select another Jetpack site that doesn't have a plan that includes VideoPress (eg add the Security plan to a new site)
* Click "Remove plan" button -> You should **NOT** see the VideoPress "benefit" card
* Manual Regression test: Visit `/media/[some site]`, flip quickly between the filters (Images, Videos) and ensure the correct data is shown (from this issue https://github.com/Automattic/wp-calypso/pull/44579)
* Manual Regression test 2: Visit `/stats/month/[some site]`, ensure the `Videos` section has data (this is an example of `<QueryMedia>` being used without a query specified.

#### Pre-merge Checklist






Related to #66076
